services/swagger/swagger.tester.js

```diff
@@ -41,3 +30,52 @@ t.create('Invalid')
     message: 'invalid',
     color: 'red',
   })
+
+t.create('Valid json 2.0')
+  .get(
+    `${getURLBase}https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/json/petstore-expanded.json`
+  )
+  .expectBadge({
+    label: 'swagger',
+    message: 'valid',
+    color: 'brightgreen',
+  })
+
+t.create('Valid yaml 3.0')
+  .get(
+    `${getURLBase}https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml`
+  )
+  .expectBadge({
+    label: 'swagger',
+    message: 'valid',
+    color: 'brightgreen',
+  })
+
+t.create('Valid with warnings')
+  .get(`${getURLBase}https://petstore3.swagger.io/api/v3/openapi.json`)
+  .expectBadge({
+    label: 'swagger',
+    message: 'valid',
+    color: 'brightgreen',
+  })
+
+// Isn't a spec, but valid json
```

Comment:
I think this is fine, as it still allows us to validate our integration with/handling of the response from the validator API 👍